packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineToggle/templateWithFacets.html

```diff
@@ -106,6 +106,7 @@
           ></c-quantic-date-facet>
         </template>
       </c-quantic-facet-manager>
+      <c-quantic-sort engine-id={engineId}></c-quantic-sort>
```

Comment:
This should not be necessary, the issue is in the Quantic Refine Modal Content component that needs to be fixed.